android/KMEA/app/src/main/assets/other-polyfills.js

```diff
@@ -14,4 +14,8 @@ if(!Array.prototype.includes){
   Array.prototype.includes = function(search){
    return !!~this.indexOf(search);
  }
-}
\ No newline at end of file
+}
+
+// This file is only referenced by the main Web engine; its "global" object is `window`.
+// `globalThis` is defined in Chrome 71+.
+var globalThis = window;
```

Comment:
Looks like we're not the only ones having to polyFill `globalThis`.
https://mathiasbynens.be/notes/globalthis

Would this stomp if `globalThis` is already set?

Reply:
It would stomp `globalThis` but that shouldn't break anything. For clarity, we could do:


```suggestion
if(typeof globalThis == ''undefined') {
  var globalThis = window;
}
```